### PR TITLE
Rithu uptime check

### PR DESCRIPTION
**Pull request by rithu1998**
Created pull request

### CODE REVIEW

**Comment by arshana** [comment]
Hi Rithu,
Your code looks great!
In the Chat class, I noticed that we both added nearly the same code to the createRootPanel method. I'm going to combine our code. There's really only a change in one line...
Other than that, we also didn't have a comment explaining the ServerInfo class. I've added one, but you could take a look at it and add to it based on your activity.

**Comment by rithu1998** [comment]
Sounds great! Yeah I noticed that too when I was reviewing your code, let me know if you have any questions on the code I wrote for the info panel 😊

File: src/codeu/chat/client/commandline/Chat.java [hunk 1]
```diff
@@ -186,6 +188,23 @@ private UserContext findUser(String name) {
...
     });
+    
+    //Added this code to respond to the user's request of info
+    panel.register("info", new Panel.Command() {
+  	  @Override
+  	  public void invoke(Scanner args) {
+  	    final ServerInfo info = context.getInfo();
+  	    if (info == null) {
+  	      // Communicate error to user - the server did not send us a valid
+  	              // info object.
```
Comment (sanjananatraj) [inline src/codeu/chat/client/commandline/Chat.java:199]: Extremely minor fix, maybe just have the comment on one line for clarity if possible? Actual code looks great.

**Comment by sanjananatraj** [comment]
Your code looks great! Your comments are clear and concise and it follows the guide really well :)

**Pull request merged**